app/views/layouts/remote_console.html.haml

```diff
@@ -2,7 +2,7 @@
 %html{:lang => I18n.locale.to_s.sub('-', '_')}
   %head
     %title
-      = _('%{product_name} HTML5 Remote Console') % {:product_name => Vmdb::Appliance.PRODUCT_NAME}
+      = _('%{vm_hostname} - %{product_name} HTML5 Remote Console') % {:vm_hostname => @vm.hostnames[0] || @vm.name, :product_name => Vmdb::Appliance.PRODUCT_NAME}
```

Comment:
I think we want this in the opposite order, so `product_name` first and then the `vm_hostname`

Reply:
The more tabs are active in the browser, the smaller the width of the tab. The `vm_hostname` is the most significant part of the tab because it differentiates each console tab from the other console tabs. That's why i believe the `vm_hostname` should be first.

Reply:
Hm, this makes sense, but I'd prefer consistency. We can ask what the others think...ping @epwinchell @dclarizio

Reply:
Hmmm . . . might be strange when I open a console for a VM named "VM1" and then the tab shows "abc-host" or something.  I do understand that the VM owner should know the VM by it's hostname as well.  It's a shame we can't put them both, but then the tab will easily be overflowed.  I'll let others chime in on this decision.

Reply:
I'd go with productname - hostname, just because of consistency with other screens.